radis/test/lbl/test_broadening.py

```diff
@@ -938,6 +938,57 @@ def test_noneq_continuum(plot=False, verbose=2, warnings=True, *args, **kwargs):
     assert res < 5.2e-6
 
 
+def test_broadening_chunksize_eq(verbose=True, plot=False, *args, **kwargs):
+    """
+    Test an equilibrium spectrum with and without chunksize,
+    using different optimization strategies
+    and ensure that the residual is very small.
```

Comment:
Can you mention this PR (where chunksize was implemented) in the docstring ? Helps future devs